### ISSUE TITLE
Occultism Foliot Lumberjack causing crash

### ISSUE THREAD

**Issue by shinji257** [EnigmaticaModpacks/Enigmatica6#4207]
### Modpack Version

0.5.25

### Describe your issue.

Upon player join server crashes with the Occulism Foliot Lumberjack either with a NPE or a ticking exception.

https://gist.github.com/shinji257/3e5268d00267b8e4c83cad4ad010ca89
https://gist.github.com/shinji257/f1badc8aa81a1cec1f9660e8e4a1fe63

### Crash Report

https://gist.github.com/shinji257/f1badc8aa81a1cec1f9660e8e4a1fe63

### Have you modified the modpack?

Yes

### User Modifications

Mods Added
blocklog-1.0.jar
itemblacklist-1.16.5-1.0.1.jar
mc2discord-forge-1.16.x-3.1.0.jar 
observable-0.1.3-forge.jar
kotlinforforge-1.15.1-obf.jar

Config Changes
advancedperipherals-common.toml - disabled chunky turtle for chunkloading
fluxnetworks-common.toml - disabled chunkloading
ftbbackups-common.toml - changed
mc2discord.toml
Mekanism/general.toml - disabled chunkloading
naturesaura-common.toml - disabled chunkloading
world/serverconfig/ftbranks/ranks.json - added
world/serverconfig/compactmachines-server.toml - disabled chunkloading
world/serverconfig/ftbchunks-server.toml - disabled offline chunkloading

### Did the issue happen in singleplayer or on a server?

Server

**Comment by shinji257** [comment]
Reproduced single player with the debug spawn egg for the Foliot Lumberjack.

```
java.lang.NullPointerException: Cannot invoke "com.github.klikli_dev.occultism.common.entity.ai.FellTreesGoal.shouldUseLumberjackDimensions()" because "this.fellTreesGoal" is null
	at com.github.klikli_dev.occultism.common.job.LumberjackJob.getDimensions(LumberjackJob.java:102)
	at com.github.klikli_dev.occultism.common.entity.spirit.SpiritEntity.lambda$getDimensions$14(SpiritEntity.java:638)
	at java.base/java.util.Optional.map(Unknown Source)
	at com.github.klikli_dev.occultism.common.entity.spirit.SpiritEntity.func_213305_a(SpiritEntity.java:638)
	at net.minecraft.entity.Entity.func_213323_x_(Entity.java:2359)
	at net.gigabit101.shrink.events.PlayerEvents.joinWorldEvent(PlayerEvents.java:83)
	at net.minecraftforge.eventbus.ASMEventHandler_31_PlayerEvents_joinWorldEvent_EntityJoinWorldEvent.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	at net.minecraft.world.server.ServerWorld.func_217440_f(ServerWorld.java:829)
	at net.minecraft.world.server.ChunkManager.func_219237_a_(ChunkManager.java:594)
	at com.mojang.datafixers.util.Either.lambda$mapLeft$0(Either.java:162)
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
	at com.mojang.datafixers.util.Either.mapLeft(Either.java:162)
	at net.minecraft.world.server.ChunkManager.func_219193_a_(ChunkManager.java:569)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(Unknown Source)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
	at net.minecraft.world.chunk.ChunkTaskPriorityQueueSorter.func_219083_b(SourceFile:58)
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144)
	at net.minecraft.world.server.ServerChunkProvider$ChunkExecutor.func_213166_h(ServerChunkProvider.java:519)
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118)
	at net.minecraft.world.server.ServerChunkProvider$ChunkExecutor.func_213168_p(ServerChunkProvider.java:527)
	at net.minecraft.world.server.ServerChunkProvider.func_217234_d(ServerChunkProvider.java:272)
	at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:722)
	at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:711)
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213161_c(SourceFile:127)
	at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.func_213186_a(MinecraftServer.java:469)
	at net.minecraft.server.MinecraftServer.func_240800_l__(MinecraftServer.java:310)
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:60)
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:621)
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232)
	at java.base/java.lang.Thread.run(Unknown Source)
```

Reproduction:
1) Load new world
2) Either spawn or summon mob
3) Exit world and reload.  Crash/stall on loading single player world.
It never finishes loading so you have to kill the pack as a whole.  It just stays at the 100% loading screen.

**Comment by shinji257** [comment]
This should be fixed in the next release of Occultism for 1.16.5

https://github.com/klikli-dev/occultism/issues/646

**Comment by NillerMedDild** [comment]
Awesome thanks!
